### PR TITLE
LPS-157273 Add test for content visibility

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:forward

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/components/layout-data-items/Container.test.js [hunk 1]
```diff
@@ -59,4 +59,16 @@ describe('Container', () => {
 		expect(link.target).toBe('_blank');
 		expect(link.textContent).toBe('Container');
 	});
+
+	it('adds content-visibility: auto when that configuration is set', async () => {
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/components/layout-data-items/Container.test.js:63]: Mr piper al aparato

Ese async no hace falta no? :trollface:

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/components/layout-data-items/Container.test.js:63]: That's true, thank you Mr Piper

**Comment by sandrodw3** [comment]
ci:stop

**Comment by sandrodw3** [comment]
ci:forward

**Pull request closed without merging**